charts/fulcio/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Fulcio
 
 type: application
 
-version: 0.2.11
+version: 0.2.12
```

Comment:
if is a breaking change I would bump it to 0.3.0 and add a readme doc to alert that, and point what things will be need to re-generated

Reply:
good advise. 
done.